spec/omniauth/strategies/bookingsync_spec.rb

```diff
@@ -26,15 +26,19 @@
     end
 
     it "returns the correct authorization url" do
-      expect(strategy.client.options[:authorize_url]).to eq "/oauth/authorize"
+      expect(strategy.client.options[:authorize_url]).to eq "oauth/authorize"
```

Comment:
that's actually an important change, might be worth documenting. What caused it?

Reply:
@Azdaroth as mentioned in a PR description, it's originating from one of underlying gems: